packages/website/blog/2022-09-03-automated-rule-docs-with-docusaurus-and-remark.md

```diff
@@ -210,7 +217,7 @@ I'm excited to focus in particular on [Docs: Proofread rule docs for clarity (#4
 
 ## Appreciation and Thanks
 
-We'd like to extend thanks to [Joshua Chen](https://github.com/Josh-Cena), one of the Docusaurus maintainers who also has been helping us with Docusaurus — and helped proofread [this blog post's PR](https://github.com/typescript-eslint/typescript-eslint/pull/5593).
+We'd like to extend thanks to [Josh Cena](https://github.com/Josh-Cena), one of the Docusaurus maintainers who also has been helping us with Docusaurus — and helped proofread [this blog post's PR](https://github.com/typescript-eslint/typescript-eslint/pull/5593).
```

Comment:
Oops, this reverts https://github.com/typescript-eslint/typescript-eslint/pull/5593/commits/236979ee1ebaf92d63f1e820be92aa169e0df4d1

Reply:
will fix!